addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1359,9 +1359,17 @@ export class Wysiwyg extends Component {
     /**
      * Stop the field changes mutation observers.
      */
-    _pauseOdooFieldObservers() {
-        for (let observerData of this.odooFieldObservers) {
-            observerData.observer.disconnect();
+    _pauseOdooFieldObservers(field) {
+        for (const fieldObserverData of this.odooFieldObservers) {
+            // Exclude inner translation fields observers. They
+            // still handle translation synchronization inside the
+            // targeted field.
+            if (
+                !fieldObserverData.field.dataset.oeTranslationSourceSha ||
+                !field.contains(fieldObserverData.field)
+            ) {
+                fieldObserverData.observer.disconnect();
+            }
```

Comment:
Actually when I read this as a whole, I'm not sure I understand what it does. The function is called `_pauseOdooFieldObservers` and receives a `field` as parameter. Therefore, I would expect it to disconnect the field observers of the provided field. However, reading the code, it seems like it does the exact opposite, that is disconnect the field observers that are **_not_** the ones from the provided field. Did I miss something? Do you see the confusion?

Either there is something that I am misunderstanding, or the parameter `field` should be renamed to something more like `excludedFields` (plural I guess since `.contains` is called on it). Either way, the documentation of the function should be updated to reflect the change.

Sorry for not noticing the issue earlier. I was thrown off by the big `filter` inside of the `for` declaration and did not think further than that.

Reply:
Hmm... what the code exactly does is disconnecting the observers of all fields except the translation fields inside the "current one". So the argument `field` is actually the current field we are editing (the "excluded fields" are the "translation" ones inside the current field itself...). Also, `contains` here is the `Node.contains()` method called on the current field element... But yes, I see the confusion here: **It looks more like a method to pause the observers of the given field.** I used `currentlyEditedField` instead and added a comment to explain the current behavior :+1: What do you think ?

Reply:
Way better, thank you! 👍